src/style.css

```diff
@@ -1 +1,16 @@
-/* styles go here */
+label {
```

Comment:
It's better not to use tag selectors, add classes to your labels and fields and use class selectors